.nvmrc

```
@@ -1 +1 @@
-16.17.0
+18.14.0
```

I suppose is better to move to the latest LTS, that is 18.16.0?
```suggestion
18.16.0
```